src/cascadia/TerminalSettingsEditor/Profiles_Base.xaml

```diff
@@ -0,0 +1,238 @@
+<!--
+    Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+    the MIT License. See LICENSE in the project root for license information.
+-->
+<Page x:Class="Microsoft.Terminal.Settings.Editor.Profiles_Base"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:Microsoft.Terminal.Settings.Editor"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:model="using:Microsoft.Terminal.Settings.Model"
+      xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="CommonResources.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+            <Style x:Key="ProfilesBaseToggleButtonStyle"
+                   TargetType="ToggleButton">
+                <Setter Property="Background" Value="{ThemeResource ExpanderHeaderBackground}" />
+                <Setter Property="MinWidth" Value="{ThemeResource FlyoutThemeMinWidth}" />
+                <Setter Property="MinHeight" Value="64" />
+                <Setter Property="BorderThickness" Value="{ThemeResource ExpanderHeaderBorderThickness}" />
+                <Setter Property="BorderBrush" Value="{ThemeResource ExpanderHeaderBorderBrush}" />
+                <Setter Property="Padding" Value="16,0,8,0" />
+                <Setter Property="Margin" Value="0,4,8,0" />
+                <Setter Property="HorizontalAlignment" Value="Stretch" />
+                <Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+                <Setter Property="VerticalAlignment" Value="Stretch" />
+                <Setter Property="HorizontalContentAlignment" Value="Left" />
+            </Style>
+        </ResourceDictionary>
+    </Page.Resources>
+
+    <!--  Use a Grid instead of a StackPanel. StackPanel suppresses the inner ScrollViewer.  -->
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <TextBlock x:Uid="Profile_BaseLayerDisclaimer"
+                   Grid.Row="0"
+                   Margin="{StaticResource StandardIndentMargin}"
+                   Style="{StaticResource DisclaimerStyle}"
+                   Visibility="{x:Bind State.Profile.IsBaseLayer}" />
+        <ScrollViewer Grid.Row="1"
+                      ViewChanging="ViewChanging">
+            <StackPanel Margin="13,0,0,0"
+                        Style="{StaticResource PivotStackStyle}">
+

```

Comment:
Do we want `Profiles_Base.xaml` to instead just have 3 buttons: `General`, `Appearance`, `Advanced`, which each have their respective pages? Instead of what it is right now where `Profiles_Base` has all the `General` settings in it, along with 2 buttons `Appearance` and `Advanced`

Reply:
Empty landing pages that just bring you to other places are a waste of clicks.

... and "General" is a waste of a page. _It's about the profile. It is general things about the profile. Generally._ They belong on the profile page.